x-pack/plugins/reporting/server/lib/store/store.ts

```diff
@@ -71,6 +71,14 @@ const jobDebugMessage = (report: Report) =>
   `[attempts: ${report.attempts}] ` +
   `[process_expiration: ${report.process_expiration}]`;
 
+const getEventLogger = (reporting: ReportingCore, report: Report) => {
+  return reporting.getEventLogger({
+    event: { id: report._id, timezone: report.payload.browserTimezone },
+    kibana: { reporting: { jobType: report.jobtype } },
```

Comment:
My main concerns here:

1. I guess it would be nice to receive the report ID in a structured way `reportId: reprot._id`.
2. Event IDs should probably be different from report IDs (`id: uuid.v1()`), as the same report can have multiple events logged associated with it.

Reply:
I think we don't need to use `event.id` at all. Each document already has the native `_id` field automatically added by ES. My guess is we should only use event.id if we have a reason to. But I like your suggestion to use the `reporting` namespace of the event data.